packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -211,7 +211,15 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 			const stringifiedTranslations = JSON.stringify( translatedStrings )
 				.replace( /"([a-z]+)":/g, '$1:' );
 
-			const outputBody = `CKEDITOR_TRANSLATIONS.add('${ language }',${ stringifiedTranslations })`;
+			const outputBody = (
+				// We need to ensure that the CKEDITOR_TRANSLATIONS variable exist and if it exists, we need to extend it.
+				'(function(d){' +
```

Comment:
Do we add this bit after the code is minified?

Reply:
Hmm, I was thinking about someone that wants to use our builds and targets ES5.  Otherwise, we can change it to arrow fn.

Reply:
I used `Object.assign` later 🤦‍♂️

Reply:
I replaced it with an arrow function. But targeting ES5 might be important, and this code won't be transpiled, it should be considered.